front/src/components/RichTextFieldRHF.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react';
+import { useController } from 'react-hook-form';
+import { Descendant } from 'slate';
+
+import {
+  RichTextEditorProps,
+  RichTextEditor,
+} from '@/components/RichTextEditor';
+import {
+  defaultErrorSerializer,
+  FieldProps,
+  RHFRegisterProps,
+  useCheckedFormRHFContext,
+} from '@/components/tailwind-ui';
+
+export type RichTextFieldRHFProps = Omit<
+  RichTextEditorProps,
+  'value' | 'onChange'
+> &
+  FieldProps &
+  RHFRegisterProps;
```

Comment:
If you update components to the latest version, you'll have to intersect with RHFValidationProps as well

Reply:
and you'll have to handle `rhfOptions.shouldUnregister`

You basically just have to forward it to `useController`

Reply:
Also with latest components version
```suggestion
  RHFControllerProps;
```